domains/min.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "minhducle25",
+    "email": "min.qut.work@gmail.com"
+  },
+  "record": {
+    "A": "51.79.135.245"
```

cannot access website also A record(s) should be in an array
```suggestion
    "A": ["51.79.135.245"]
```